htdocs/fourn/facture/paiement.php

```diff
@@ -424,8 +435,8 @@
 	        // Bouton Enregistrer
 	        if ($action != 'add_paiement')
 	        {
-				print '<br><div class="center"><input type="checkbox" checked name="closepaidinvoices"> '.$langs->trans("ClosePaidInvoicesAutomatically");
-				print '<br><input type="submit" class="button" value="'.$langs->trans('Save').'"></div>';
+				print '<center><br><input type="checkbox" checked="checked" name="closepaidinvoices"> '.$langs->trans("ClosePaidInvoicesAutomatically");
```

Comment:
Balise <center> are deprecated in html5. Please leave the div balise with the class "center".